UnitTestProject/UnitTest1.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Internship1.Controllers;
+using Internship1.Service;
+using Internship1.Models;
+using System.Web.Mvc;
+using Azure.Storage.Blobs.Models;
+
+namespace UnitTestProject
+{
+    [TestClass]
+    public class UnitTest1
```

Comment:
Rename UnitTest1 class in something better. 
Create UnitTest class for each service that you test